### DISCUSSION TITLE
有能直接加载 s3mb 格式的组件么

### DISCUSSION THREAD

**Discussion by yingnian** [zouyaoji/vue-cesium#570] [Q&A]
当前通过如下方式添加
`function getLayerMedaData(viewer: any, Cesium: any) {
	const url = baseUrl + replaceUrl
	axios.get(url).then(res => {
		if (res.status === 200 && res.data.length > 0) {
			res.data.forEach((item: any) => {
				addSceneS3mLayer(
					{
						url: baseUrl + item.path + '/config.scp'
					},
					viewer,
					Cesium
				)
			})
		}
	})
}
function addSceneS3mLayer(options: any, viewer: any, Cesium: any) {
	viewer.scene.addS3MTilesLayerByScp(options.url)
}`
但是在左右平移的时候 发现3维模型与实际的底图产生了偏移

**Comment by zouyaoji** [comment]
没有哈，你可以咨询一下超图的技术
